examples/siro_sandbox/hb_config_helper.py

```diff
@@ -40,52 +39,26 @@ def update_config_and_args(
             )
             agent_key = "" if len(sim_config.agents) == 1 else "agent_0_"
             agent_sensor_name = f"{agent_key}third_rgb"
-            args.debug_images.append(agent_sensor_name)
+            hitl_config.debug_images.append(agent_sensor_name)
             gym_obs_keys.append(agent_sensor_name)
 
-        # Code below is ported from interactive_play.py. I'm not sure what it is for.
-        if True:
-            if "pddl_success" in task_config.measurements:
-                task_config.measurements.pddl_success.must_call_stop = False
-            if "rearrange_nav_to_obj_success" in task_config.measurements:
-                task_config.measurements.rearrange_nav_to_obj_success.must_call_stop = (
-                    False
-                )
-            if "force_terminate" in task_config.measurements:
-                task_config.measurements.force_terminate.max_accum_force = -1.0
-                task_config.measurements.force_terminate.max_instant_force = (
-                    -1.0
-                )
-
-        if args.never_end:
```

Comment:
This logic is more cleanly handled directly through the config system. E.g. `rearrange` and `pick_throw_vr` apps specify `max_episode_steps: 0` because it makes sense for those applications' episodes to never end.